scm-ui/ui-components/src/navigation/index.ts

```diff
@@ -36,3 +36,4 @@ export { default as SecondaryNavigationItem } from "./SecondaryNavigationItem";
 export { default as ExternalLink } from "./ExternalLink";
 export { default as ExternalNavLink } from "./ExternalNavLink";
 export { default as useNavigationLock } from "./useNavigationLock";
+export { default as LoginButton } from "./LoginButton";
```

Comment:
I'm note sure but it looks like we have the possibility to replace the logout button as well (ep: `primary-navigation.logout`). Should we move the logout button and export it too?